README.md

```diff
@@ -1,2 +1,5 @@
 # picATrip
 picATrip - Beyond 05, team 2 project
+
+A platform for sharing experiences from trips in a form of photos based on various parameters as locations/duration time/difficulty, and provides a trip search functionality and commenting functionality. 
+It allows users to have the oppurtunity to contribute their trip experience and thoughts as well as learn about trips of others across the country, and adding comment if desired.
```

Comment:
I would advise to use some markdown formats to the content
https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet
Such as:
What is picAtrip?
Who are we?